src/main/java/echoserver/EchoServer.java

```diff
@@ -13,18 +16,18 @@ public static void main(String[] args) throws IOException {
         final Integer SPECIFIED_PORT = validator.parsePort(args);
 
         ServerSocket hostSocket = new ServerSocket(SPECIFIED_PORT);
-        Host hostServer = new Host(hostSocket);
+        ClientHub hostServer = new ClientHub(hostSocket);
```

Comment:
What is a `ClientHub`?

Reply:
I renamed `Host` to `ClientHub` because I thought it was more descriptive? Like, it's a hub for clients to connect to. Is the name confusing? Should I change it?